### DISCUSSION TITLE
Expo Splashscreen Glitch?? (Maybe)

### DISCUSSION THREAD

**Discussion by RainPlays09** [expo/expo#33305] [Archived: Questions]
is it bug that if i set the icon property into my `app.json`, it uses this image overwriting the `splash` and `android.splash` property??

**Comment by veris-kamaraj** [comment]
same issue it shows the icon only not the splash image and after the js bundle loaded it shows the splash image for second only and disappears

**Reply by RainPlays09** [reply]
I hope they fix this issue
